torchci/lib/bot/rolloutUtils.ts

```diff
@@ -63,8 +63,8 @@ const landCheckPilotGroup = new Set([
   "HaoZeke",
   "amjames",
   "fdrocha",
-]);
+];
 
 export function isInLandCheckAllowlist(username: string) {
-  return landCheckPilotGroup.has(username);
+  return landCheckPilotGroup.some(u => u.toLowerCase() === username.toLowerCase());
```

Comment:
Not that it matters too much here, but it changes the time complexity from amortized constant to linear.

Reply:
Yea I thought about that and didn't think it mattered for the <100 here. I feel like since GH usernames are supposed to be case insensitive, this is less fragile, but I don't really mind either way.

Reply:
Wouldn't that just work? (and make sure that `landCheckPilotGroup` is only initialized with lowercase usernames
```suggestion
  return landCheckPilotGroup.has(username.toLowerCase());
```